test/housekeeping_test.js

```diff
@@ -402,33 +402,35 @@ describe('House Keeper', () => {
       ]
     });
 
-    describeVolumesStub.withArgs(ec2[region], 'describeVolumes', houseKeeper.describeVolumesParams)
-      .returns({
-        Volumes: [{
-          Attachments: [],
-          AvailabilityZone: region, 
-          CreateTime: new Date().toString(), 
-          Size: 8, 
-          SnapshotId: "snap-1234567890abcdef0", 
-          State: "in-use", 
-          VolumeId: "vol-049df61146c4d7901", 
-          VolumeType: "standard",
-        }]
-      });
+    describeVolumesStub.withArgs(sinon.match(function(value) {
+      return value === ec2['us-west-2'] 
+    })).returns({
+       Volumes: [{
+         Attachments: [],
+         AvailabilityZone: 'us-west-2', 
+         CreateTime: new Date().toString(), 
+         Size: 8, 
+         SnapshotId: "snap-1234567890abcdef0", 
```

Comment:
This is probably a minor quibble... SnapshotID and VolumeID are probably unique IDs so we shouldn't be using the same values for both volumes in this test.

We can probably even extract out the returned Volumes object in each call to us-westVolumes and us-eastVolumes. In our future PRs, we could then use those Volume objects in our future tests.

Reply:
I changed only the last character in both, out of sheer laziness. We should probably make them more unique. :stuck_out_tongue_closed_eyes:

Reply:
And I was too lazy to notice that they were actually different strings. 😆

Reply:
lazy<sup>lazy</sup>

Reply:
:)